themes/ddbasic/gulpfile.js

```diff
@@ -81,13 +81,38 @@ gulp.task('sass', 'Process SCSS using libsass',
   }
 );
 
+/**
+ * Usage:
+ * gulp ksssass
+ *
+ * Precompile all sass files into one single bundle.css file in the sass_css folder.
+ * This all in one file is used by kss for the style sheet.
+ *
+ */
+gulp.task('ksssass', 'Process SCSS into one css file using libsass',
+  function () {
+    gulp.src(sassPath)
+      .pipe(sass({outputStyle: 'compact'})
+        .on('error', sass.logError))
+      // Save verbose output for testing purposes.
+      .pipe(gulp.dest('./sass_css_verbose_output'))
```

Comment:
The verbose output does not seem to be related to KSS. If that is the case then it should not be a part of the task.

Reply:
Removed.